packages/visualizations-stories/src/vizualisations/1-bar-chart.stories.tsx

```diff
@@ -105,8 +105,8 @@ const BarChart = <Name extends string>({
   const colorScale = useColorScale(data, colorCursors, palette);
 
   return (
-    <div style={{ display: "inline-block" }}>
-      <Legend data={data} colorScale={colorScale} cursors={colorCursors}/>
+    <div style={{ width: 420 }}>
+      <Legend data={data} colorScale={colorScale} cursors={colorCursors} itemWidth={colorCursors.length * 80} style={{ height: 40, overflow: "scroll" }}/>
```

Comment:
```suggestion
      <Legend data={data} colorScale={colorScale} cursors={colorCursors} itemWidth={colorCursors.length * 80} style={{ height: 40, overflowY: "scroll" }}/>
```

should produce only one scrollbar